core/api.go

```diff
@@ -408,7 +424,7 @@ func (a *api) getFileConfig(g *gin.Context, mill m.Mill, use string, plaintext b
 	} else {
 		var file *pb.FileIndex
 		var err error
-		reader, file, err = a.node.FileData(use)
+		reader, file, err = a.node.FileContent(use)
```

Comment:
this should be simplified via `:=` as well

Reply:
Using `:=` would create a new local var, whereas here we mean the `reader` defined at the beginning of the function.

Reply:
Good point.